core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -1784,8 +1784,7 @@ private long loadMetadataAndJournal(LockedInodePath inodePath, LoadMetadataOptio
     UnderFileSystem ufs = resolution.getUfs();
     try {
       if (!ufs.exists(ufsUri.toString())) {
-        throw new FileDoesNotExistException(
-            ExceptionMessage.PATH_DOES_NOT_EXIST.getMessage(path.getPath()));
+        return AsyncJournalWriter.INVALID_FLUSH_COUNTER;
```

Comment:
This happens often when people use MUST_CACHE and call list_status on a directory. What I learned from "effective-java" (Item 57) is that exceptions should not be used for normal control flow which can be slow.  I can change this to return a different Exception (say FileDoesNotExistInUFSException) if you prefer. Then I can remove the logging based on this.